packages/yoroi-extension/chrome/constants.js

```diff
@@ -53,6 +53,7 @@ export function genCSP(request: {|
 
   // Analytics
   connectSrc.push('https://analytics.emurgo-rnd.com/');
+  connectSrc.push('https://*.amplitude.com');
```

Comment:
Required to be in the `manisfest.json` otherwise the requests will fail due to `CSP`